README.md

```diff
@@ -537,7 +537,7 @@ You can add your own assertions using `expect.extend` and `expect.assert`:
 expect.extend({
   toBeAColor() {
     expect.assert(
-      this.actual.match(/^#[a-fA-F0-9]{6}$/),
+      this.actual.match(/^#[a-f0-9]{3,6}$/i),
```

Comment:
the case-insensitivity actually makes this regex slower, so it's intentional to have `[a-fA-F0-9]`.

As for the `3,6`, can you add test cases that require this change?